app/assets/scripts/components/connected/personnel-table.js

```diff
@@ -149,13 +149,18 @@ class PersonnelTable extends SFPComponent {
         label: <SortHeader id='emer' title='Emergency' sort={this.state.table.sort} onClick={''} /> // for filtering options check .../api/v2/personnel/?limit=2 - the Filters button, Ordering
       }];
 
+      // Should add the other types if needed
+      const typeLongNames = {
```

Comment:
Does this RR type come from a mapping on the backend or so? Can we add a quick comment pointing to the code in the backend if it does?

Reply:
It's here but I guess this is for the dropdown on the Django Admin only: https://github.com/IFRCGo/go-api/blob/e92b0ceadd70297a574fe4410d76eb7bf8614411/deployments/models.py#L98

Reply:
Yup, if you can add a comment in the javascript code, something like:

```
// These types reference types defined in the backend models here: https://github.com/IFRCGo/go-api/blob/e92b0ceadd70297a574fe4410d76eb7bf8614411/deployments/models.py#L98-L106
```

Will just help if someone in the future needs to look-up where the `rr` value comes from. Just a code comment though :-) - the implementation looks fine to me.